component/site/views/default/helpers/defaultloadedfromtemplate.php

```diff
@@ -1027,14 +1027,14 @@ function DefaultLoadedFromTemplate($view, $template_name, $event, $mask, $templa
 						}
 						if (stripos($fieldval, "%d") !== false)
 						{
-							$days = intval($timedelta / (60 * 60 * 24));
+							$days = (int) $timedelta / (60 * 60 * 24);
```

Comment:
This is really pedantic and will have zero impact on performance - for a modern compiler/interpreter it will go through exactly the same underlying code.  All these changes do is make commits take longer to merge :(